### ISSUE TITLE
[BUG] Companion 2.4.0 (2.4.0+4878-76646782) "Export Page" function broken.

### ISSUE THREAD

**Issue by jjbutler83** [bitfocus/companion#2210]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Just updated my Raspberry Pi Companion build to 2.4.0+4878-76646782 and immediately have an issue where the "Export Page"  function, located on the "Buttons" page above the grid layout is no longer working and produces the following error:

TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at EventEmitter.<anonymous> (/usr/local/src/companion/lib/loadsave.js:872:28)
    at EventEmitter.emit (events.js:412:35)
    at /usr/local/src/companion/lib/server_express.js:96:11
    at Layer.handle [as handle_request] (/usr/local/src/companion/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/usr/local/src/companion/node_modules/express/lib/router/index.js:328:13)
    at /usr/local/src/companion/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/usr/local/src/companion/node_modules/express/lib/router/index.js:346:12)
    at next (/usr/local/src/companion/node_modules/express/lib/router/index.js:280:10)
    at textParser (/usr/local/src/companion/node_modules/body-parser/lib/types/text.js:69:7)

### Steps To Reproduce

Try to export any single one page from the "Export page" button on the "Buttons page."
The Full Config Import and Export options, located in their own area, seem to be working fine still.

### Expected Behavior

Need to be able to export single button pages as previously able in previous builds.

### Environment (please complete the following information)

```markdown
- OS: Windows 11 Pro and Mac OS 13.0.1
- Browser: Firefox, Chrome, MS Edge, Safari (Mac OS)
- Companion Version: 2.4.0+4878-76646782
```

### Additional context

The same version of Companion installed on a Windows 11 Pro machine does NOT exhibit this issue and appears to be only linked to the Raspberry Pi build version.

**Comment by Julusian** [comment]
This appears to be an issue when older configs are upgraded.

If you need to export a page, try enabling rotary actions on one of the buttons on the page (and optionally disabling it straight away)

This will be fixed in 2.4.1

**Comment by jjbutler83** [comment]
Thanks for the info!

Sure enough, the rotary actions toggle did the trick and will suffice as a work around for now until 2.4.1.

Thanks again!




[A picture containing logo<br><br>Description automatically generated]



Justin Butler

Freelance Video Specialist

J. Butler Production Services, LLC

C: (407) 448-7476

E: ***@***.***

W: https://jb-ps.com<https://jb-ps.com/>

Social Media Portal<https://linktr.ee/jjbutler83>


________________________________
